src/oidcop/session/claims.py

```diff
@@ -119,7 +113,7 @@ def get_claims(self, session_id: str, scopes: str, usage: str) -> dict:
 
     def get_claims_all_usage(self, session_id: str, scopes: str) -> dict:
         _claims = {}
-        for usage in ["userinfo", "introspection", "id_token", "access_token"]:
+        for usage in self.claims_types:
             _claims[usage] = self.get_claims(session_id, scopes, usage)
```

Comment:
This should be wrapped in a try except:
```python
try:
    _claims[usage] = self.get_claims(session_id, scopes, usage)
except ServiceError:
    pass
```
OR `_get_module` must not raise and return `None`

Reply:
None has a special meaning. Which is that the `usage` doesn't come with any claims release.
An example of such a `usage` is authorization_code.

ServiceError on the other hand means: you are asking for claims connected to a specific `usage` I should know about but don't or can't handle.